apps/www/registry/new-york/ui/calendar.tsx

```diff
@@ -20,48 +19,85 @@ function Calendar({
+        months:
+          "relative flex flex-col sm:flex-row space-y-4 sm:space-x-4 sm:space-y-0",
         month: "space-y-4",
-        caption: "flex justify-center pt-1 relative items-center",
-        caption_label: "text-sm font-medium",
-        nav: "space-x-1 flex items-center",
-        nav_button: cn(
-          buttonVariants({ variant: "outline" }),
-          "h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100"
-        ),
-        nav_button_previous: "absolute left-1",
-        nav_button_next: "absolute right-1",
-        table: "w-full border-collapse space-y-1",
-        head_row: "flex",
-        head_cell:
-          "text-muted-foreground rounded-md w-8 font-normal text-[0.8rem]",
-        row: "flex w-full mt-2",
-        cell: cn(
-          "relative p-0 text-center text-sm focus-within:relative focus-within:z-20 [&:has([aria-selected])]:bg-accent [&:has([aria-selected].day-outside)]:bg-accent/50 [&:has([aria-selected].day-range-end)]:rounded-r-md",
-          props.mode === "range"
-            ? "[&:has(>.day-range-end)]:rounded-r-md [&:has(>.day-range-start)]:rounded-l-md first:[&:has([aria-selected])]:rounded-l-md last:[&:has([aria-selected])]:rounded-r-md"
-            : "[&:has([aria-selected])]:rounded-md"
+        nav: "flex items-center justify-between absolute w-full z-10 px-1",
+        button_previous: cn(
+          buttonVariants({
+            variant: "outline",
+            className:
+              "h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100",
+          })
         ),
-        day: cn(
-          buttonVariants({ variant: "ghost" }),
-          "h-8 w-8 p-0 font-normal aria-selected:opacity-100"
+        button_next: cn(
+          buttonVariants({
+            variant: "outline",
+            className:
+              "h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100",
+          })
         ),
-        day_range_start: "day-range-start",
-        day_range_end: "day-range-end",
-        day_selected:
-          "bg-primary text-primary-foreground hover:bg-primary hover:text-primary-foreground focus:bg-primary focus:text-primary-foreground",
-        day_today: "bg-accent text-accent-foreground",
-        day_outside:
-          "day-outside text-muted-foreground opacity-50  aria-selected:bg-accent/50 aria-selected:text-muted-foreground aria-selected:opacity-30",
-        day_disabled: "text-muted-foreground opacity-50",
-        day_range_middle:
-          "aria-selected:bg-accent aria-selected:text-accent-foreground",
-        day_hidden: "invisible",
+        month_caption: "flex justify-center items-center h-7",
+        caption_label: "text-sm font-medium",
+        month_grid: "border-collapse space-y-1",
+        weekdays: "flex",
+        weekday: "text-muted-foreground w-9 font-normal text-xs",
+        weeks: "",
+        week: "flex mt-2",
+        day: "p-0",
+        outside: "bg-accent/40",
```

Comment:
```suggestion
        outside: "opacity-40",
```
simply applying opacity instead of apply the `bg-accent` again make it looks better on custom theming
| bg-accent/40                                                                              | opacity-40                                                                                |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
| ![image](https://github.com/user-attachments/assets/05935262-a4c4-4cf1-9ea6-698865a92451) | ![image](https://github.com/user-attachments/assets/c0b033b5-1cff-4a9b-828a-bd60dda916c1) |